options.lua

```diff
@@ -14,6 +14,7 @@ defaultOptions = {
 		dontClickReleases = false,
 		useSoulstoneRez = true,
 		dontAcceptInvite = false,
+		dontPostAuctions = false,
```

Comment:
I'd default this to true, since this is one of the more dangerous ones to leave active.